packages/@react-spectrum/s2/intl/de-DE.json

```diff
@@ -22,6 +23,6 @@
   "table.sortDescending": "Absteigend sortieren",
   "tag.actions": "Aktionen",
   "tag.hideButtonLabel": "Weniger zeigen",
-  "tag.noTags": "Keine",
+  "tag.noTags": "Kein(e)",
```

Comment:
?

Reply:
I'm reverting this, change leveraged from another app, but no linguistic reason for change in React-Spectrum